objects/src/accounts/account_id.rs

```diff
@@ -140,8 +144,46 @@ impl AccountId {
         ];
         let mut current_digest = Self::compute_digest(current_seed, code_root, storage_root);
 
+        #[cfg(feature = "log")]
+        {
+            let init_seed: Vec<u8> = init_seed.iter().cloned().flatten().collect();
+            let init_seed: String = to_hex(&init_seed).expect("hex formatting failed");
+            log::info!(
+                "Generating new account seed [pow={}, current_digest={}, current_seed={}, init_seed={}]",
+                digest_pow(current_digest),
+                digest_hex(current_digest),
+                word_hex(current_seed),
+                init_seed,
```

Comment:
I added the `hex` encoding mostly because I expected to be able to see the `0`s as PoW went on. I noticed later that the PoW is not done in the left/right most bits, so it is hard to see them. I kept the hex formatted version because it is easier to parse visually, and added a `pow` count with the number of `0` bits.